install/agones/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
the [`make do-release`](https://github.com/GoogleCloudPlatform/agones/blob/master/build/Makefile#L229) will need to be changed so that it zips up the install dir and grabs both the install.yaml and the helm chart.